lib/flagsmith/engine/segments/models.rb

```diff
@@ -86,6 +87,12 @@ def match_modulo_value(trait_value)
           false
         end
 
+        def match_in_value(trait_value)
+          return @value.split(',').any? { |x| x == trait_value.to_s } if trait_value.is_a?(String) || trait_value.is_a?(Integer)
```

Comment:
Perhaps there is a reason that we've implemented it this way, but could we do something like the following instead? Seems a little but neater.  

```ruby
return @value.split(',').include?(trait_value.to_s)
```

Reply:
Yeah, much nicer. Done!